### PR TITLE
[FIX] support spins on armel armhf arm64

### PR DESCRIPTION
**Pull request by mr-c**
Partially addresses https://github.com/seqan/seqan3/issues/1381

### CODE REVIEW

File: include/seqan3/core/parallel/detail/spin_delay.hpp [hunk 1]
```diff
@@ -72,6 +75,10 @@ class spin_delay
     {
         #if defined(__SSE2__)  // AMD and Intel
             _mm_pause();
+	#elif defined(__armel__) || defined(__ARMEL__) // armel, but broken
```
Comment (rrahn) [inline include/seqan3/core/parallel/detail/spin_delay.hpp]: it seems the alignment to the other statements is off by 2 spaces

Comment (rrahn) [inline include/seqan3/core/parallel/detail/spin_delay.hpp]: Why is this not covered by the final else block? Or does __armel__ also define __arm__ but not the other way around?

Comment (mr-c) [reply include/seqan3/core/parallel/detail/spin_delay.hpp]: Arggh.. spaces vs. tabs :-P

Comment (mr-c) [reply include/seqan3/core/parallel/detail/spin_delay.hpp]: Dropped

**Pull request merged**